database.hh

```diff
@@ -327,10 +333,10 @@ struct table_stats {
     int64_t live_sstable_count = 0;
     /** Estimated number of compactions pending for this column family */
     int64_t pending_compactions = 0;
-    int64_t partition_creations = 0;
-    int64_t partition_reuses = 0;
-    int64_t row_creations = 0;
-    int64_t row_overwrites = 0;
+    int64_t memtable_partition_insertions = 0;
+    int64_t memtable_partition_hits = 0;
+    int64_t memtable_row_insertions = 0;
+    int64_t memtable_row_hits = 0;
```

Comment:
These should have been introduced in this patch, not in the struct rename patch.